exporters/otlp/include/opentelemetry/exporters/otlp/otlp_http_client.h

```diff
@@ -49,6 +49,9 @@ struct OtlpHttpClientOptions
 {
   std::string url;
 
+  /** SSL options. */
+  ext::http::client::HttpSslOptions ssl_options;
```

Comment:
this can be under macro ENABLE_OTLP_HTTP_SSL_PREVIEW ?

Reply:
Fixed